extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -53,15 +47,17 @@ export class SchemaCompareDialog {
 	private targetDbEditable: string;
 	private previousSource: mssql.SchemaCompareEndpointInfo;
 	private previousTarget: mssql.SchemaCompareEndpointInfo;
+	private initDialogComplete: Deferred<void>;
```

Comment:
thanks for adding this! Can you also add this to the options dialog?

Reply:
FYI - this is probably useful to have as a base abstract class things can extend from so it's easy to add to dialogs as you see fit. See https://github.com/microsoft/azuredatastudio/blob/main/extensions/arc/src/ui/components/initializingComponent.ts as an example

Reply:
That is a great example, Charles. 
I will update the options dialog in another PR and will introduce the base abstract class thing there (when it will actually be needed to be done at two places). So for now, kept it as-is.